src/fluent_wrapper.rs

```diff
@@ -0,0 +1,86 @@
+use std::collections::HashMap;
+use std::error::Error;
+use std::fs::File;
+use std::fs::read_dir;
+use std::io::BufReader;
+use std::io::prelude::*;
+use std::io::Result;
+use std::path::Path;
+
+use fluent_bundle::{FluentBundle, FluentResource};
+
+use i18n::I18NProvider;
+
+pub fn read_from_file<P: AsRef<Path>>(filename: P) -> Result<Vec<FluentResource>> {
+    let mut file = try!(File::open(filename));
+    let reader = BufReader::new(file);
+
+    let mut fluent_resources = Vec::new();
```

Comment:
This part is unnecessary because `FluentResource` can handle multiple pairs. Only a full read of the file is needed, and the whole string passed to construct `FluentResource`.